# Certificate configuration scenarios.md

```diff
@@ -0,0 +1,246 @@
+# Certificate configuration scenarios
+
+There are five scenarios/use cases for configuring a certificates.
+Use the applicable certificate configuration scenario according to your needs.
+
+* [Scenario 1: Use a PKCS12 (keystore) with Zowe generated certificates](#scenario-1-use-a-pkcs12-keystore-with-zowe-generated-certificates)
+* [Scenario 2: Use a PKCS12 (keystore) and import a certificate generated by another CA](#scenario-2-use-a-pkcs12-keystore-and-import-a-certificate-generated-by-another-ca)
+* [Scenario 3: Use a z/OS Keyring with Zowe generated certificates](#scenario-3-use-a-zos-keyring-with-zowe-generated-certificates)
+* [Scenario 4: Use a z/OS Keyring and connect to an existing certificate](#scenario-4-use-a-zos-keyring-and-connect-to-an-existing-certificate)
+* [Scenario 5: Use a z/OS Keyring and import a certificate stored in a data set](#scenario-5-use-a-zos-keyring-and-import-a-certificate-stored-in-a-data-set)
+
+## Scenario 1: Use a PKCS12 (keystore) with Zowe generated certificates
+
+1. Select the type of certificate storage with `type: PKCS12`  
+Possible values include the following values:  
+
+    * PKCS12  
```

Comment:
Is this list useful? If not, I suggest it be omitted.

Reply:
Scenario 1 implies PKCS12. The rest of the list here would not be possible.